develop/dev-guide-timeouts-in-tidb.md

```diff
@@ -12,9 +12,15 @@ aliases: ['/zh/tidb/dev/timeouts-in-tidb']
 
 TiDB 的事务的实现采用了 MVCC（多版本并发控制）机制，当新写入的数据覆盖旧的数据时，旧的数据不会被替换掉，而是与新写入的数据同时保留，并以时间戳来区分版本。TiDB 通过定期 GC 的机制来清理不再需要的旧数据。
 
-默认配置下 TiDB 可以保障每个 MVCC 版本（一致性快照）保存 10 分钟，读取时间超过 10 分钟的事务，会收到报错 `GC life time is shorter than transaction duration`。
+- 使用 4.0 之前版本的 TiDB：
 
-当用户确信自己需要更长的读取时间时，比如在使用了 Dumpling 做全量备份的场景中（当 Dumpling 备份的是一致性的快照），可以通过调整 TiDB 中`mysql.tidb` 表中的 `tikv_gc_life_time` 的值来调大 MVCC 版本保留时间，需要注意的是 tikv_gc_life_time 的配置是立刻影响全局的，调大它会为当前所有存在的快照增加生命时长，调小它会立即缩短所有快照的生命时长。过多的 MVCC 版本会拖慢 TiKV 的处理效率，在使用 Dumpling 做完全量备份后需要及时把 `tikv_gc_life_time` 调整回之前的设置。
+    默认配置下 TiDB 可以保障每个 MVCC 版本（一致性快照）保存 10 分钟，读取时间超过 10 分钟的事务，会收到报错 `GC life time is shorter than transaction duration`。
+
+- 使用 4.0 及之后版本的 TiDB：
+
+    正在运行的事务，如果持续时间不超过 24 小时，在运行期间 GC 会被阻塞，不会出现 `GC life time is shorter than transaction duration` 报错。
+
+当用户确信自己需要更长的读取时间时，比如在使用了 Dumpling 做全量备份的场景中（当 Dumpling 备份的是一致性的快照），可以通过调整 TiDB 中 `mysql.tidb` 表中的 `tikv_gc_life_time` 的值来调大 MVCC 版本保留时间，需要注意的是 `tikv_gc_life_time` 的配置是立刻影响全局的，调大它会为当前所有存在的快照增加生命时长，调小它会立即缩短所有快照的生命时长。过多的 MVCC 版本会拖慢 TiKV 的处理效率，在使用 Dumpling 做完全量备份后需要及时把 `tikv_gc_life_time` 调整回之前的设置。
```

Comment:
From v5.0 the system variable `tidb_gc_life_time` is introduced, while directly updating `mysql.tidb` table becomes unrecommended. Please consider update it to using system variable `tidb_gc_life_time`, and better add a hyperlink to the document of the variable.

Reply:
Thanks for advising. I changed it to describe them separately. PTAL.